code/src/index.css

```diff
@@ -1,13 +1,153 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: "Roboto Mono", monospace;
+  font-weight: 600;
+  box-sizing: border-box;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: var(--clr-white);
+  display: flex;
+  justify-content: center;
+  align-items: center;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: "Roboto Mono", monospace;
+}
+
+:root {
+  --clr-black: #111111;
+  --clr-white: #fff;
+  --clr-primary: #f7f1f1;
+  --clr-secondary: #d1c4c4;
+  --clr-pink: #f8a6a6;
+}
+.main {
+  max-width: 600px;
+  margin: auto;
+}
+
+h1 {
+  font-family: "Shrikhand", cursive;
+  font-size: 40px;
+  letter-spacing: 1px;
+  text-align: center;
+}
+.input-wrap,
+.thought-card {
```

Comment:
on your .thought-card try to add the following code so the CSS doesn't break if the message is too long to fit!

```
    overflow: hidden;
    overflow-wrap: break-word;
```